packages/lib-sourcify/src/Compilation/auxdataUtils.ts

```diff
@@ -54,90 +57,124 @@ function getDiffPositions(original: string, modified: string): number[] {
 /**
  *   Checks the raw bytecode indeed includes the auxdata diff at the given position
  */
-function bytecodeIncludesAuxdataDiffAt(
-  bytecode: string,
-  auxdataDiff: AuxdataDiff,
+function bytecodeIncludesCompilerAuxdataDiffAt(
+  originalBytecode: string,
+  compilerAuxdataDiff: CompilerAuxdataDiff,
   position: number,
 ): boolean {
-  const { real, diffStart } = auxdataDiff;
+  const { real, diffStart } = compilerAuxdataDiff;
   // the difference (i.e metadata hash) starts from "position". To get the whole auxdata instead of metadata go back "diffStart" and until + "real.length" of the auxdata.
-  const extracted = bytecode.slice(
+  const extracted = originalBytecode.slice(
     position - diffStart,
     position - diffStart + real.length,
   );
   return extracted === real;
 }
 
-function getAuxdatasDiff(originalAuxdatas: string[], editedAuxdatas: string[]) {
-  const auxdataDiffs: AuxdataDiff[] = [];
-  for (let i = 0; i < originalAuxdatas.length; i++) {
+function getCompilerAuxdatasDiffs(
+  auxdatasFromCompiler: string[],
+  editedAuxdatasFromCompiler: string[],
+) {
+  const compilerAuxdataDiffs: CompilerAuxdataDiff[] = [];
+  for (let i = 0; i < auxdatasFromCompiler.length; i++) {
     const diffPositions = getDiffPositions(
-      originalAuxdatas[i],
-      editedAuxdatas[i],
+      auxdatasFromCompiler[i],
+      editedAuxdatasFromCompiler[i],
     );
-    auxdataDiffs.push({
-      real: originalAuxdatas[i],
+    compilerAuxdataDiffs.push({
+      real: auxdatasFromCompiler[i],
       diffStart: diffPositions[0],
-      diff: originalAuxdatas[i].substring(
+      diff: auxdatasFromCompiler[i].substring(
```

Comment:
I see the `diff` field used to be from `originalAuxdatas` but below in the diagram we show it on the edited part. Either one of them is false

Reply:
The `diffStart` also seems to be wrong in that case